NetCord/Embed.cs

```diff
@@ -1,24 +1,81 @@
 ﻿namespace NetCord;
 
+/// <summary>
+/// Displays embedded content such as an image or URL, alongside a title and various other fields. You can only have up to 10 embeds per message, and the total text of all embeds must be less than 6000 characters.
+/// </summary>
 public class Embed : IJsonModel<JsonModels.JsonEmbed>
 {
     JsonModels.JsonEmbed IJsonModel<JsonModels.JsonEmbed>.JsonModel => _jsonModel;
     private readonly JsonModels.JsonEmbed _jsonModel;
-
+    
+    /// <summary>
+    /// The text that is placed above the description, usually highlighted. Also directs to a URL if given, has a 256 character limit.
+    /// </summary>
     public string? Title => _jsonModel.Title;
+    
+    /// <summary>
+    /// The type of the embed, always set to <see cref="EmbedType.Rich"/> for webhook embeds.
```

Comment:
I think webooks can also send non rich embeds by sending a URL. I am not sure why it is documented as here in the official documentation though

Reply:
Yeah that was my source for it, I'll change it if that isn't the case though